types/system-task/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for system-task 1.0
+// Project: https://github.com/leocwlam/system-task
+// Definitions by: Leo Lam <https://github.com/leocwlam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
+
+/// <reference types="node" />
+
+export function asyncProcess(items: any[], executeAsyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
+export function syncProcess(items: any[], executeSyncCall: any, task: SystemTask, errors: any[]): Promise<any>;
+
+export default class SystemTask {
```

Comment:
This package was declared using `module.exports = ...` or some variant. It should be declared with an `export = ...` like one of the following:

### Variables

```ts
declare var xyz: SomeType;

export = xyz;
```

### Classes

```ts
class Foo {
  // ...
}

export = Foo
```

### Functions

```ts
function foo(): any;

export = foo;
```

To import the entity using a default import, you will need to enable `esModuleInterop`. Alternatively, you can just import using a more appropriate syntax like

```ts
import foo = require("foo");
```

[See more here in the FAQ](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9f9f6430117d025aef62ed56128deb65508edc86/README.md#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).